content/en/docs/tasks/administer-cluster/endpointslice.md

```diff
@@ -0,0 +1,55 @@
+---
+reviewers:
+- bowei
+- freehan
+title: Enabling Endpoint Slices
+content_template: templates/task
+---
+
+{{% capture overview %}}
+This page provides an overview of enabling Endpoint Slices in Kubernetes.
+{{% /capture %}}
+
```

Comment:
Would it make sense to name this file: endpoint-slice.md?

Reply:
Updated, thanks!